packages/graphql-config-utilities/src/config.ts

```diff
@@ -31,31 +36,26 @@ export function resolveSchemaPath(
   // schemaPath is nullable in graphq-config even though it cannot actually be
   // omitted. This function simplifies access to the schemaPath without
   // requiring a type guard.
-  if (!project.schemaPath) {
+  if (!project.schema) {
     // this case should never happen with a properly formatted config file.
     // graphql-config currently does not perform any validation so it's possible
     // for a mal-formed schema to be loaded at runtime.
-    throw new Error(
-      `Missing GraphQL schemaPath for project '${resolveProjectName(project)}'`,
-    );
+    throw new Error(`Missing GraphQL schemaPath for project '${project.name}'`);
   }
 
   // resolve fully qualified schemaPath
-  const schemaPath = project.resolveConfigPath(project.schemaPath);
-
+  const schemaPath = resolve(project.dirpath, project.schema as string);
   if (ignoreMissing) {
     return schemaPath;
   }
 
   if (!existsSync(schemaPath)) {
-    const forProject = project.projectName
-      ? ` for project '${project.projectName}'`
-      : '';
+    const forProject = project.name ? ` for project '${project.name}'` : '';
     throw new Error(
       [
         `Schema not found${forProject}.`,
         `Expected to find the schema at '${schemaPath}' but the path does not exist.`,
-        `Check '${project.configPath}' and verify that schemaPath is configured correctly${forProject}.`,
+        `Check '${project.filepath}' and verify that schemaPath is configured correctly${forProject}.`,
```

Comment:
```suggestion
        `Check '${project.filepath}' and verify that schema is configured correctly${forProject}.`,
```

I'm making these suggestions without knowing we still retain "schemaPath" at the config level.

Reply:
Yeah given the api changes of graphql-config 3 I think we might be able to drop it entirely. Some of the changes felt awkward. ie going out of my way not to break the api.

Reply:
`.graphqlconfig` can still have `schemaPath` this then becomes `schema` on the object instance.